lib/grape-swagger/endpoint.rb

```diff
@@ -207,19 +207,20 @@ def response_object(route, options)
 
         next build_file_response(memo[value[:code]]) if file_response?(value[:model])
 
-        response_model = @item
-        response_model = expose_params_from_model(value[:model]) if value[:model]
-
         if memo.key?(200) && route.request_method == 'DELETE' && value[:model].nil?
           memo[204] = memo.delete(200)
           value[:code] = 204
+          next
         end
 
-        next if value[:code] == 204
-        next unless !response_model.start_with?('Swagger_doc') && (@definitions[response_model] || value[:model])
+        # Explicitly request no model with { model: '' }
+        next if value[:model] == ''
```

Comment:
mmh … ok … will have a look on it later, maybe it can be get together with no model key given

Reply:
@LeFnord It did that because by default if no model key is provided ([line 219](https://github.com/ruby-grape/grape-swagger/pull/804/files#diff-7af39528837333c960830618fb6a52e9R219)) the model use for the response will be the one returned by `GrapeSwagger::DocMethods::PathString#build` called here https://github.com/ruby-grape/grape-swagger/blob/08b8c2fe0518043e4a479453327068ab0285d6b9/lib/grape-swagger/endpoint.rb#L100